### PR TITLE
certified-update

### PR DESCRIPTION
**Pull request by stevegraham**
An executable that downloads a current cert bundle and replaces the
old one. Keep current people!

### CODE REVIEW

File: bin/certified-update [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env ruby
+
+require 'open-uri'
+require 'pathname'
+require 'certified'
+
+CERT_BUNDLE_URL = 'https://raw.githubusercontent.com/bagder/ca-bundle/master/ca-bundle.crt'
```
Comment (stevegraham) [inline bin/certified-update:7]: @badger do you mind if this pulls from your ca-bundle repo?

**Comment by stevegraham** [comment]
cc @martoche @pangloss @zkwentz @bitzesty @imajes @phlegx @althafhameez @PBruce

File: bin/certified-update (hunk 1 above)
Comment (stevegraham) [inline bin/certified-update:7]: \* @bagder

Comment (bagder) [inline bin/certified-update:7]: Not at all, it is made to get used!

Comment (stevegraham) [inline bin/certified-update:7]: Thanks!

**Pull request merged**